scrapers/src/json_converter.ts

```diff
@@ -34,6 +34,8 @@ export const parseRequirement = (
       return parseRequiredRange(req);
     case "COURSE":
       return parseRequiredSimple(req);
+    case "CREDITS":
+      throw new Error("unimplemented!");
```

Comment:
just to make `yarn tsc` work

Reply:
woah, did yarn tsc not work before or is this related to some change made in this pr??

Reply:
didn't work before

Reply:
crazy